vm/error.go

```diff
@@ -1,18 +1,68 @@
 package vm
 
-type ErrorClass struct {
-	*BaseClass
+import (
+  "fmt"
+)
+
+var (
+  // UndefinedMethodErrorClass ...
+  UndefinedMethodErrorClass *RError
+  // ArgumentErrorClass ...
+  ArgumentErrorClass *RError
+)
+
+func initErrors() {
+  bc := &BaseClass{Name: "UndefinedMethodError", Methods: newEnvironment(), ClassMethods: newEnvironment(), Class: classClass, pseudoSuperClass: objectClass, superClass: objectClass}
+  UndefinedMethodErrorClass = &RError{BaseClass: bc}
+  bc = &BaseClass{Name: "ArgumentError", Methods: newEnvironment(), ClassMethods: newEnvironment(), Class: classClass, pseudoSuperClass: objectClass, superClass: objectClass}
+  ArgumentErrorClass = &RError{BaseClass: bc}
+}
+
+// RError ...
+type RError struct {
+  *BaseClass
 }
 
+// Error ...
 type Error struct {
-	Class   *ErrorClass
-	Message string
+  Class   *RError
```

Comment:
Actually I'm thinking using `*RClass` is fine. Because we may not do any performance optimization on errors so we don't need a special class type for it. Besides using `*RClass` means users can create inherit their own exception classes like in Ruby.